pkg/azure/api/api.go

```diff
@@ -39,12 +40,11 @@ import (
 	"github.com/Azure/go-autorest/autorest/to"
 )
 
-const (
-	userAgent = "cilium"
+var (
+	log       = logging.DefaultLogger.WithField(logfields.LogSubsys, "azure-api")
+	userAgent = fmt.Sprintf("cilium/%s", version.Version)
```

Comment:
The version string already contains `cilium`. Wouldn't `userAgent = version.Version` be sufficient?

Reply:
@rolinh -- https://github.com/cilium/cilium/blob/master/pkg/version/version.go#L55 this tells me otherwise

Reply:
Oh indeed, you're right :man_facepalming: